lib/dal-test/src/helpers.rs

```diff
@@ -37,7 +37,7 @@ pub fn generate_fake_name() -> Result<String> {
 #[allow(clippy::expect_used)]
 #[macro_export]
 macro_rules! connection_annotation_string {
-    ($str:expr) => {
+    ($str:expr_2021) => {
```

Comment:
Interesting... we may want to clean these up later

Reply:
Yeah, this was the result of running `cargo fix --edition` so is perhaps a backwards compat thing